<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-11-27T14:29:45Z" name="Suite" finished-at="2023-11-27T14:30:15Z" duration-ms="30146">
    <groups>
    </groups>
    <test started-at="2023-11-27T14:29:45Z" name="Test" finished-at="2023-11-27T14:30:15Z" duration-ms="30146">
      <class name="com.selenium.testcases.Search">
        <test-method is-config="true" signature="startChromeBrowser()[pri:0, instance:com.selenium.testcases.Search@dd0c991]" started-at="2023-11-27T19:59:47Z" name="startChromeBrowser" finished-at="2023-11-27T20:00:01Z" duration-ms="14754" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startChromeBrowser -->
        <test-method signature="searchWithValidProduct()[pri:1, instance:com.selenium.testcases.Search@dd0c991]" started-at="2023-11-27T20:00:01Z" name="searchWithValidProduct" finished-at="2023-11-27T20:00:03Z" duration-ms="1487" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- searchWithValidProduct -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.selenium.testcases.Search@dd0c991]" started-at="2023-11-27T20:00:03Z" name="tearDown" finished-at="2023-11-27T20:00:04Z" duration-ms="918" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="startChromeBrowser()[pri:0, instance:com.selenium.testcases.Search@dd0c991]" started-at="2023-11-27T20:00:04Z" name="startChromeBrowser" finished-at="2023-11-27T20:00:11Z" duration-ms="7365" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startChromeBrowser -->
        <test-method signature="searchWithInvalidProduct()[pri:2, instance:com.selenium.testcases.Search@dd0c991]" started-at="2023-11-27T20:00:11Z" name="searchWithInvalidProduct" finished-at="2023-11-27T20:00:13Z" duration-ms="1448" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Prduct is found expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Prduct is found expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at com.selenium.pages.SearchPage.verifySearchFailedWarning(SearchPage.java:35)
	at com.selenium.testcases.Search.searchWithInvalidProduct(Search.java:35)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- searchWithInvalidProduct -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.selenium.testcases.Search@dd0c991]" started-at="2023-11-27T20:00:14Z" name="tearDown" finished-at="2023-11-27T20:00:15Z" duration-ms="1406" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method signature="searchWithoutProduct()[pri:3, instance:com.selenium.testcases.Search@dd0c991]" started-at="2023-11-27T20:00:15Z" name="searchWithoutProduct" finished-at="2023-11-27T20:00:15Z" depends-on-methods="com.selenium.testcases.Search.searchWithInvalidProduct" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method Search.searchWithoutProduct()[pri:3, instance:com.selenium.testcases.Search@dd0c991] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method Search.searchWithoutProduct()[pri:3, instance:com.selenium.testcases.Search@dd0c991] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1084)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- searchWithoutProduct -->
      </class> <!-- com.selenium.testcases.Search -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
